pagopa/io-app
# chore(Federated Identity Management System): [IAFIMS-6] Implements FIMS webview component and screens

## Short description
This PR implements the screen to handle SSO on FIMS system through a webview

## How to test
1. Enables FIMS local FF
2. execute from anywhere:
```typescript
        handleInternalLink(
          dispatch,
          `ioit://${FIMS_ROUTES.WEBVIEW}?url=http://127.0.0.1:3000/myportal_playground.html`
        );
```
check the cookie is correctly passed out.
